.github/workflows/build.yml

```diff
@@ -57,24 +57,32 @@ jobs:
 
   linux-build-all:
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version : ['1.16', '1.17']
     steps:
     - name: checkout
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
 
-    - name: setup golang
+    - name: setup golang ${{ matrix.go-version }}
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16
+        go-version: ${{ matrix.go-version }}
 
     - name: set up QEMU
       uses: docker/setup-qemu-action@master
       with:
         platforms: all
 
-    - name: build.all rook
-      run: tests/scripts/github-action-helper.sh build_rook_all
+    - name: build.all rook with go ${{ matrix.go-version }}
+      run: |
+        if [[ "${{ matrix.go-version }}" == "1.17" ]]; then
+          sed -i 's/go 1.16/go 1.17/g' go.mod
```

Comment:
Why does this need to change? I believe we want to keep it to 1.16 so the go modules will be pulled in according to the least supported version.

Reply:
Good point that's not needed for now., let's just test the build on 1.17 only without pulling deps.